.github/copilot-instructions.md

```diff
@@ -0,0 +1,46 @@
+# Copacetic - Container Image Vulnerability Patching
+
+## Project Overview
+Copacetic (Copa) is a CLI tool that patches container image vulnerabilities using BuildKit. It applies OS package updates directly to container images without requiring full image rebuilds.
+
+**Main command**: `copa patch -i IMAGE [-r REPORT] -t TAG`
+**Module**: `github.com/project-copacetic/copacetic`
+
+## Folder Structure
+- `pkg/patch/`: CLI commands and core patching logic
+- `pkg/buildkit/`: BuildKit integration and platform discovery
+- `pkg/pkgmgr/`: Package manager adapters (dpkg, rpm, apk)
+- `pkg/report/`: Vulnerability report parsing
+- `pkg/types/`: Type definitions and configurations
+- `main.go`: Root CLI setup with Cobra framework
+
+## Libraries and Frameworks
+- BuildKit for container image manipulation
+- Cobra for CLI framework
+- Trivy as default vulnerability scanner
+- Logrus for structured logging
+- Go Container Registry libraries for image operations
+
+## Coding Standards
+- Follow Go best practices and `golangci-lint` rules
+- Use structured logging with `logrus`
+- Implement proper error wrapping with `fmt.Errorf`
+- Write comprehensive tests for new functionality
```

Comment:
should we specify integration and unit tests? and maybe we can add adding relevant documentation for new functionality